resources/backend/api_endpoints.md

```diff
@@ -0,0 +1,35 @@
+# API Endpoints
+API endpoints are essentially points of communication in which a user can interact with and utilize an API's functions. In the context of this project, they are ways for frontend users to interact with and use backend functions.
+![API Endpoint Diagram](../images/api_endpoint_diagram.png)
+Imagine the backend is an island with multiple ship ports. If a frontend user wants to interact with the backend, they send a ship to one of the ports to either deliver, retrieve, or modify information. These ships are called **HTTP requests**.
+## What does an endpoint look like?
```

Comment:
Looks good! I like the island analogy! To be more clear (explicit), an incoming ship is an HTTP request while an outgoing ship is the HTTP response. Each of the ship ports is an HTTP endpoint.